### DISCUSSION TITLE
С запретом некоторые сайты лагают

### DISCUSSION THREAD

**Discussion by kekicheska** [bol-van/zapret#844] [Q&A]
Скачал запрет, вроде все ок, дискорд не пингует, но при включении демки она появляется через секунды 3, мелочь но не приятно, при запуске некоторых сайтов, сайт грузит очень долго, но другие же сайты грузят очень быстро, нужен хелп!

**Comment by kekicheska** [comment]
P.S также сообщение в дискорде грузит тоже с задержкой (секунды 2)

**Comment by Fenrir-0xFF** [comment]
У меня с конфигом для https норм всё робит по крайней мере на проводном билайне --filter-tcp=443 --wssize=1:6 --dpi-desync=disorder2 

В hostlist auto добавь домены: 
discord.com
discord.gg
discordapp.com
discordapp.net
discord.app
discord.media
discordcdn.com
discord.dev
discord.new
discord.gift
discordstatus.com
dis.gd
discord.co
discord-attachments-uploads-prd.storage.googleapis.com

Насчёт голосовых чятов как там сейчяс ХЗ - не чекал, но пару недель назад эти настройки работали --filter-udp=50000-50099 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-fake-quic=/files/fake/quic_initial_google_com.bin
